user-guide/Advanced_Functionality/Databases/Cassandra_TWCS/Cassandra_TWCS.md

```diff
@@ -40,4 +38,4 @@ The following recommended limitations apply:
 | Day trending       | 30 days          | 1500 days               |
 
 > [!NOTE]
-> These recommended maximum TTL values cannot be changed.
\ No newline at end of file
+> Regardless of how the Cassandra database is configured, the recommended maximum TTL values cannot be changed.
```

Comment:
In this section we talk about 'recommended max TTL' while with the note we give the impression it can't be changed which sound like a contradiction. If we don't want users to configure a TTL higher than what is in the table we should remove the 'recommended' and say something like this:
'Regardless of how the Cassandra database is configured, the TTL of the tables should never exceed the maximum TTL as defined in this table.'
@MariekeGO feel free to correct my phrasing